File: examples.py
from pbtGenerator import pbtGenerator

# Example Usage
myPBT = pbtGenerator({"templateName": 'EpicPythonGeneratedTemplate#2'})
myPBT.addMesh('testMesh', 'sm_cube_002', {"position": {"X" : 0, "Y": 99, "Z" : 0}})
myPBT.addMesh('otherMesh', 'sm_cube_002', {"rotation" : {"X" : 100, "Y" : 360, "Z" : 10}})
myPBT.addMesh('nextMesh', 'sm_cube_002', {"scale" : {"X" : 99999, "Y" : 20, "Z" : 10}})
print(myPBT.generatePBT())

File: examples.js
import {pbtGenerator} from './pbtGenerator';

let pbt = new pbtGenerator({templateName: 'EpicJavascriptTemplate', })
pbt.addMesh('myMesh', 'sm_cube_002', {})
pbt.addMesh('SecondMesh', 'sm_cube_002', {position: {X: 100, Y: 40, Z: 200}})
pbt.addMesh('anotherMeshForShowcase', 'sm_cube_002', {rotation: {Pitch: 100, Yaw: 100, Roll: 50}})
console.log(pbt.generatePBT())